src/components/calcite-color-picker/readme.md

```diff
@@ -2,6 +2,29 @@
 
 <!-- Auto Generated Below -->
 
+## Usage
+
+### Basic
+
+```html
+<calcite-color-picker dir="ltr" scale="m" value="#b33f33" appearance="default"></calcite-color-picker>
+```
+
+### Minimal
+
+For a minimal design, you can hide unused color formats and options:
+
+```html
+<calcite-color-picker
+  appearance="minimal"
+  allow-empty=""
```

Comment:
These boolean attributes shouldn't have values assigned to them right?

Reply:
That was my thought as well. However in [storybook](https://esri.github.io/calcite-components/?path=/story/components-controls-colorpicker--simple) if you check the 'hide-saved' box and go to the HTML snippet it has the empty string. So that's what I went with. I can change it and log an issue for the storybook if it shouldn't have a value.

Reply:
Yeah, that's kind of a known storybook limitation but lets remove it from our docs.

Reply:
Fixed it. Thanks for clarifying, storybook had me a bit confused there.